docs/features/software-templates/writing-templates.md

```diff
@@ -304,7 +321,8 @@ You might have noticed in the examples that there are `{{ }}`, and these are a
 `yaml` together. All the form inputs from the `parameters` section, when passed
 to the steps will be available by using the template syntax
 `{{ parameters.something }}`. This is great for passing the values from the form
-into different steps and reusing these input variables.
+into different steps and reusing these input variables. To pass arrays or objects
+use the syntax `{{ json paramaters.something }}`.
```

Comment:
Maybe just clarify that `parameters.something` should be of type `object` or `array` in the `jsonSchema` here too?

Reply:
Like `nicknames` above for example. Or something like that?